packages/flutter/lib/src/material/flutter_logo.dart

```diff
@@ -0,0 +1,76 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+
+import 'icon_theme.dart';
+import 'icon_theme_data.dart';
+import 'colors.dart';
+
+/// The Flutter logo, in widget form. This widget respects the [IconTheme].
+///
+/// See also:
+///
+///  * [IconTheme], which provides ambient configuration for icons.
+///  * [Icon], for showing icons the Material design icon library.
+///  * [ImageIcon], for showing icons from [AssetImage]s or other [ImageProvider]s.
+class FlutterLogo extends StatelessWidget {
```

Comment:
Interesting that this is in material.  I guess all the color theming is worth it.